private/doc/openapi-3-v2.0.json

```diff
@@ -0,0 +1,209 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "description": "Sources Internal",
+    "version": "2.0.0",
+    "title": "Sources Internal",
+    "contact": {
+      "email": "support@redhat.com"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "security": [
+    {
+      "UserSecurity": [
+
+      ]
+    }
+  ],
+  "paths": {
+  },
```

Comment:
It really looks like the generator isn't working with this, the `paths` object should contain all of the `/internal` endopints - so lets just create one manually rather than modifying the generator.

Reply:
For the most part the rest of the schema looks fine - so maybe just manually add the paths to this?

Reply:
I agree here.  If we can't trust the generator to do the right thing, let's just do a manual generation of the schema.

Reply:
It was created manually so I have to add the /paths (at least it brokes the specs)